docs/control-structures.rst

```diff
@@ -29,8 +29,13 @@ something like::
         }
     }
 
-Input parameters can be used just as any other local variable
-can be used, they can also be assigned to.
+Input parameters can be used as any other local variable and they can also be assigned to.
```

Comment:
@chriseth I'd like to clarify this, but need to check something. Does it mean they can have values assigned?

Reply:
Yes they can be assigned to.